stats/DataAggregation.md

```diff
@@ -0,0 +1,67 @@
+# Data Aggregation API Overview
+The stats library allows users to define measurements how data are aggregated and broken down.
+The core data types used are:
+* `Aggregation`: defines the aggregation type (e.g. `Distribution`, `Sum`, `Mean`).
+* `View`: defines how individual measurements are broken down by tags and aggregated.
+
+## Aggregation
+An Aggregation describes how data collected is aggregated.
+
+The library SHOULD provide support for multiple types of Aggregations:
+* `Count`: indicates that data collected and aggregated with this `Aggregation` type will be turned
+into a count value.
+* `Sum`: indicates that data collected and aggregated with this `Aggregation` will be summed up.
+* `Mean`: indicates that data collected and aggregated with this `Aggregation` will calculate the
+mean value.
+* `Distribution`: indicates that the desired `Aggregation` is a histogram distribution. A
```

Comment:
Stackdriver Distribution includes mean, sum of squared deviation, and range; C++ and Java include those in their Distribution type. To reflect existing implementations I would make that "Distribution", and possibly include a histogram-only "Histogram" type.

Reply:
That should get into the DistributionData. The data that we produce is not part of this API.